external-crates/move/crates/move-compiler/src/command_line/compiler.rs

```diff
@@ -143,18 +146,37 @@ impl Compiler {
                         .map(|(k, v)| (k.into(), v))
                         .collect::<NamedAddressMap>(),
                 );
+                let paths: Vec<Symbol> = if vfs_root.is_none() {
+                    let paths: Vec<_> = paths.into_iter().map(|p| p.into()).collect();
+                    let paths: Vec<_> = paths.iter().map(|p| p.as_str()).collect();
+                    find_filenames_and_keep_specified(&paths, &mut find_move_files)?
```

Comment:
In general, I want to get the compiler out of the business of finding files. It is a relic from before the package system.  But its functionality is relied on a bit too much to rip out from this PR, though I totally can if y'all want me to.

Reply:
I'm a very large fan of ripping it out as soon as possible.